examples/MicroserviceExample/Utils/Messaging/MessageReceiver.cs

```diff
@@ -98,7 +98,7 @@ private IEnumerable<string> ExtractTraceContextFromBasicProperties(IBasicPropert
             }
             catch (Exception ex)
             {
-                this.logger.LogError(ex, "Failed to extract trace context: {ex}");
+                this.logger.LogError(ex, "Failed to extract trace context.");
```

Comment:
I removed the extra parameter from the message template to matches the LogError at line 85.